apps/web/pages/apps/installed.tsx

```diff
@@ -223,40 +223,34 @@ export default function IntegrationsPage() {
   const { t } = useLocale();
   const query = trpc.useQuery(["viewer.integrations", { onlyInstalled: true }]);
   return (
-    <Shell
-      heading={t("installed_apps")}
-      subtitle={t("manage_your_connected_apps")}
-      large
-      customLoader={<SkeletonLoader />}>
-      <AppsShell>
-        <QueryCell
-          query={query}
-          success={({ data }) => {
-            return data.items.length > 0 ? (
-              <>
-                <IntegrationsContainer variant="conferencing" />
-                <CalendarListContainer />
-                <IntegrationsContainer variant="payment" className="mt-8" />
-                <IntegrationsContainer variant="other" className="mt-8" />
-                <Web3Container />
-              </>
-            ) : (
-              <EmptyScreen
-                Icon={Icon.FiGrid}
-                headline={t("empty_installed_apps_headline")}
-                description={
-                  <>
-                    <span className="mb-6 block">{t("empty_installed_apps_description")}</span>
-                    <Button href="/apps" EndIcon={Icon.FiArrowRight}>
-                      {t("empty_installed_apps_button")}
-                    </Button>
-                  </>
-                }
-              />
-            );
-          }}
-        />
-      </AppsShell>
-    </Shell>
+    <AppsLayout heading={t("installed_apps")} subtitle={t("manage_your_connected_apps")}>
+      <QueryCell
+        query={query}
+        success={({ data }) => {
+          return data.items.length > 0 ? (
+            <>
+              <IntegrationsContainer variant="conferencing" />
+              <CalendarListContainer />
+              <IntegrationsContainer variant="payment" className="mt-8" />
+              <IntegrationsContainer variant="other" className="mt-8" />
```

Comment:
NIT: while we currently refactor this page, should we rename all "Integrations" to "Apps"